python/pshexs/memcached.py

```diff
@@ -0,0 +1,34 @@
+
+import pymemcache
+from pshconfig import Config
```

Comment:
Hm, did the library use the namespace pshconfig?  We always spell out the company name in package names, I think.  Or at least we should.

Reply:
This rename was a suggestion from Nick, so we can change that. For right now, the imports are set up as `pshconfig`, but they could be `platformshconfig` or something.